save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/RegistrationView.kt

```diff
@@ -0,0 +1,267 @@
+        userInfo.name.let { fieldsMap[InputTypes.USER_NAME] = it }
+    }
+
+    @Suppress(
+        "TOO_LONG_FUNCTION",
+        "EMPTY_BLOCK_STRUCTURE_ERROR",
+        "LongMethod",
+        "MAGIC_NUMBER",
+        "PARAMETER_NAME_IN_OUTER_LAMBDA",
+    )
+    override fun ChildrenBuilder.render() {
+        if (state.userInfo?.isActive == false) {
+            ReactHTML.main {
+                className = ClassName("main-content mt-0 ps")
+                ReactHTML.div {
+                    className = ClassName("page-header align-items-start min-vh-100")
+                    ReactHTML.span {
+                        className = ClassName("mask bg-gradient-dark opacity-6")
+                    }
+                    ReactHTML.div {
+                        className = ClassName("row justify-content-center")
+                        ReactHTML.div {
+                            className = ClassName("col-sm-4")
+                            ReactHTML.div {
+                                className = ClassName("container card o-hidden border-0 shadow-lg my-2 card-body p-0")
+                                ReactHTML.div {
+                                    className = ClassName("p-5 text-center")
+                                    ReactHTML.h1 {
+                                        className = ClassName("h4 text-gray-900 mb-4")
+                                        +"Registration new user"
+                                    }
+                                    ReactHTML.label {
+                                        ReactHTML.input {
+                                            type = InputType.file
+                                            hidden = true
+                                            onChange = { event ->
+                                                postImageUpload(event.target)
+                                            }
+                                        }
+                                        ariaLabel = "Change organization's avatar"
+                                        ReactHTML.img {
+                                            className =
+                                                    ClassName("avatar avatar-user width-full border color-bg-default rounded-circle")
+                                            src = state.image?.path?.let {
+                                                "/api/$v1/avatar$it"
+                                            }
+                                                ?: run {
+                                                    "img/undraw_profile.svg"
+                                                }
+                                            height = 260.0
+                                            width = 260.0
+                                        }
+                                    }
+                                    ReactHTML.form {
+                                        className = ClassName("needs-validation")
```

Comment:
Not sure if we need this because it should already be inside `inputTextFormRequired`

Reply:
Removed it